src/mdx-pages/guides/react.mdx

```diff
@@ -28,14 +28,17 @@ export const VideoJS = (props) => {
       if (!videoElement) return;
 
       const player = playerRef.current = videojs(videoElement, options, () => {
-        player.log('player is ready');
+        videojs.log('player is ready');
         onReady && onReady(player);
       });
 
-    // You can update player in the `else` block here, for example:
+    // You could update an existing player in the `else` block here
+    // on prop change, for example:
     } else {
-      player.autoplay(options.autoplay);
-      player.src(options.sources);
+      // const player = playerRef.current;
+
+      // player.autoplay(options.autoplay);
+      // player.src(options.sources);
```

Comment:
I believe this is the big fix in this PR. Any reason to use `videojs.log` over `player.log`, it seems like that should work.

Reply:
`player` isn't defined everywhere it was used, so just made it `videojs.log` to keep it simple.

Reply:
isn't it defined on line 30?

Reply:
oh, I see, sorry I was conflating the examples into a single one. Yeah, I guess it's fine, it isn't a huge difference either way.